src/components/organisms/Table/stories/v2.stories.tsx

```diff
@@ -0,0 +1,306 @@
+  const [currentData, setCurrentData] = React.useState<TableRow[]>(userData as TableRow[])
+
+  return (
+    <Table sortData={currentData} originalData={userData} setSortData={setCurrentData}>
+      <Head>
+        <Row>
+          <Cell type='head' value='full name' field='employee' sortable color='success' />
+          <Cell type='head' value='january' field='january' />
+          <Cell type='head' value='february' field='february' />
+          <Cell type='head' value='march' field='march' />
+          <Cell type='head' value='april' field='april' />
+          <Cell type='head' value='may' field='may' />
+          <Cell type='head' value='june' field='june' />
+          <Cell type='head' value='july' field='july' />
+          <Cell type='head' value='august' field='august' />
+          <Cell type='head' value='september' field='september' />
+          <Cell type='head' value='october' field='october' />
+          <Cell type='head' value='november' field='november' />
+          <Cell type='head' value='december' field='december' />
+          <Cell type='head' value='average (2020)' field='average' color='warning' />
+        </Row>
+      </Head>
+      <Body>
+        {currentData.map((row: TableRow) => {
+          return (
+            <Row key={row.id.toString()}>
+              <Cell value={row['employee']} color='success' />
+              <Cell value={row['january']} />
+              <Cell value={row['february']} />
+              <Cell value={row['march']} />
+              <Cell value={row['april']}/>
+              <Cell value={row['may']}/>
+              <Cell value={row['june']} />
+              <Cell value={row['july']} />
+              <Cell value={row['august']} />
+              <Cell value={row['september']} />
+              <Cell value={row['october']} />
+              <Cell value={row['november']} />
+              <Cell component={<Badge value='n/a' />} />
+              <Cell value={row['average']} color='warning' />
+            </Row>
+          )
+        })}
+      </Body>
+      <Foot>
+        {metaData.map((row: TableRow) => {
+          return (
+            <Row key={row.id.toString()}>
+              <Cell value={row['employee']} color='error' />
+              <Cell value={row['january']} color='light' />
+              <Cell value={row['february']} color='light' />
+              <Cell value={row['march']} color='light' />
+              <Cell value={row['april']} color='light' />
+              <Cell value={row['may']} color='light' />
+              <Cell value={row['june']} color='light' />
+              <Cell value={row['july']} color='light' />
+              <Cell value={row['august']} color='light' />
+              <Cell value={row['september']} color='light' />
+              <Cell value={row['october']} color='light' />
+              <Cell value={row['november']} color='light' />
+              <Cell component={<Badge value='n/a' />} color='light' />
```

Comment:
I guess it's not needed to have a `component` prop where we can just use a `children`.

Reply:
yeah, we can do that